### ISSUE TITLE
'dict' object has no attribute 'myget'

### ISSUE THREAD

**Issue by pansism** [hectornieto/pyTSEB#49]
Hi @hectornieto!

First of all, I'd like to thank you for releasing your code.

I tried to run `TSEB_local_image_main.py` but I got the following error: 

```
Exception has occurred: AttributeError
'dict' object has no attribute 'myget'

  File "/mnt/c/Users/User/Projects/pyTSEB/pyTSEB/TSEBConfigFileInterface.py", line 163, in _parse_common_config
    conf['model'] = parser.myget('model')
  File "/mnt/c/Users/User/Projects/pyTSEB/pyTSEB/TSEBConfigFileInterface.py", line 243, in get_data
    conf = self._parse_common_config(parser)
  File "/mnt/c/Users/User/Projects/pyTSEB/TSEB_point_time_series_main.py", line 26, in run_TSEB_from_config_file
    setup.get_data(config_data, is_image=False)
  File "/mnt/c/Users/User/Projects/pyTSEB/TSEB_point_time_series_main.py", line 41, in <module>
    run_TSEB_from_config_file(config_file)
```

After some troubleshooting, I found out that the cause is `parse_input_config()`, which returns `dict(parser.items("top"))`.

If I change the return statement to `return parser`, everything works perfectly.

**Comment by hectornieto** [comment]
Thanks a lot for your comment @pansism. We have now [reverted the latest commit](https://github.com/hectornieto/pyTSEB/commit/669a8bac8b83d6c6e1ea009efc7728ea6535db56), which was causing that error

All the best,

**Comment by pansism** [comment]
Thanks @hectornieto!

**Comment by radosuav** [comment]
Should be fixed with [7845f14](https://github.com/hectornieto/pyTSEB/commit/7845f14fc03d9b676d94c627e052fc13c77f776d). Please reopen if not.
